urumanifest/constants.py

```diff
@@ -63,6 +63,8 @@ class _manifests(NamedTuple):
 
     # Legacy -- to be deleted??? -- TransGaming Cider Wrapper (macOS)
     "mac": _manifests(None, None, "macExternal"),
+    "macInternal": _manifests("MacThinInternal", None, "MacInternal"),
+    "macExternal": _manifests("MacThinExternal", None, "MacExternal"),
```

Comment:
Are these legacy manifests?

Reply:
macInternal and macExternal are current. Should they be declared as something else?

Reply:
They should probably be listed before the comment that says legacy manifests follow.